addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -364,7 +364,11 @@ var ListRenderer = BasicRenderer.extend({
             isPassword: 'password' in node.attrs,
         });
         this._handleAttributes($td, node);
-        return $td.html(formattedValue).attr('title', formattedValue);
+        var title = formattedValue;
+        if (field.type === 'boolean') {
+            title = field_utils.format.boolean(value, field, {forceString: true});
+        }
```

Comment:
Not sure the title is necessary in the case of boolean fields. Title is important for trimmed values, which will never be the case for checkboxes. Moreover, 'True' and 'False' is a bit technical, so I'd put no title for booleans:
`var title = field.type !== 'boolean' ? formattedValue : '';`

Reply:
I agree. The commit has been updated :wink: